samples/basic/threads/README.rst

```diff
@@ -22,7 +22,8 @@ Each thread is then defined at compile time using K_THREAD_DEFINE.
 Building
 ********
 
-.. code-block:: console
-
-    $ cd samples/basic/threads
-    $ make
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/threads
+   :board: 96b_carbon
```

Comment:
Might want to add a follow-up patch that makes clear it's an example for this board, the same as done elsewhere in this PR.